packages/@ember/-internals/owner/index.ts

```diff
@@ -55,25 +55,26 @@ export const OWNER = symbol('OWNER');
   `audioType` passed as an attribute:
 
   ```app/components/play-audio.js
-  import Component from '@ember/component';
+  import Component from '@glimmer/component';
   import { computed } from '@ember/object';
   import { getOwner } from '@ember/application';
 
   // Usage:
   //
-  //   {{play-audio audioType=model.audioType audioFile=model.file}}
+  //   <PlayAudio @audioType={{model.audioType}} @audioFile={{model.file}}/>
   //
-  export default Component.extend({
-    audioService: computed('audioType', function() {
+  export default class extends Component {
+    @computed('audioType')
```

Comment:
Ah, sorry, one more change: Remove the `@computed`, it shouldn't be necessary with Glimmer components (and it would have to point to `args.audioType` anyways)

Reply:
oh didn't know that. So cool! 👌